example/simple.dart

```diff
@@ -122,7 +122,7 @@ Future<HintResults> _dartCompleterAsync(CodeMirror editor,
   List list = _numbers.where((s) => s.startsWith(word)).toList();
 
   return new Future.delayed(new Duration(milliseconds: 200), () {
-    return new HintResults.fromStrings(list,
+    return new HintResults.fromHints(list,
```

Comment:
This one _is_ actually a list of strings. I should put a parameter on the type so the analyze can help out here -